src/config.rs

```diff
@@ -2467,19 +2467,15 @@ pub struct BackupStreamConfig {
     #[online_config(skip)]
     pub num_threads: usize,
     #[online_config(skip)]
-    pub io_threads: usize,
-    #[online_config(skip)]
     pub enable: bool,
     #[online_config(skip)]
     pub temp_path: String,
     #[online_config(skip)]
-    pub temp_file_size_limit_per_task: ReadableSize,
+    pub file_size_limit: ReadableSize,
     #[online_config(skip)]
     pub initial_scan_pending_memory_quota: ReadableSize,
     #[online_config(skip)]
     pub initial_scan_rate_limit: ReadableSize,
-    #[online_config(skip)]
-    pub use_checkpoint_v3: bool,
```

Comment:
... for keep test cases valid, I would prefer to use the [skip](https://serde.rs/field-attrs.html#skip) directive to prevent users setting it but removing it directly.